src/ui/widget-forms/ContentConfigFormBody.js

```diff
@@ -20,6 +20,15 @@ const maxLength70 = maxLength(70);
 
 export const MultipleContentsConfigContainerId = `widgets.${MULTIPLE_CONTENTS_CONFIG}`;
 
+export const validateFieldValueForFormikValidators = (value, validators) => {
+  let error = null;
+  validators.forEach((validator) => {
+    const validationError = validator(value);
+    if (validationError) error = validationError;
+  });
+  return error;
+};
+
```

Comment:
maybe you can now replace this function with the one that we have in formikUtils